dashboards-observability/public/components/visualizations/charts/metrics/metrics.tsx

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { EuiFlexItem, EuiFlexGroup } from '@elastic/eui';
+import React from 'react';
+import { sum, min, max, meanBy } from 'lodash';
+import { EmptyPlaceholder } from '../../../event_analytics/explorer/visualizations/shared_components/empty_placeholder';
+import './metrics.scss';
+
+export const Metrics = ({ visualizations }: any) => {
+  const {
+    data,
+    metadata: { fields },
+  } = visualizations.data.rawVizData;
+  const { dataConfig = {}, layoutConfig = {} } = visualizations.data.userConfigs;
+  const dataConfigTab =
+    visualizations.data?.rawVizData?.metrics?.dataConfig &&
+    visualizations.data.rawVizData.metrics.dataConfig;
```

Comment:
Take a pull of the latest code with valueOptions and check whether this still works or not.